.github/workflows/internal-ci.yaml

```diff
@@ -51,21 +51,21 @@ jobs:
       version: ${{ needs.test_version.outputs.version }}
 
   test_extended_test:
-    needs: [test_version]
     uses: ./.github/workflows/p2p-workflow-extended-test.yaml
     secrets:
       env_vars: |
         TEST_VARIABLE=value
     with:
       dry-run: true
-      version: ${{ needs.test_version.outputs.version }}
+      main-branch: ${{ github.ref }}
```

Comment:
so its fully executed?

Reply:
yea, otherwise the prod and extended tests are skipped, even with dry run